tokens/utilities/index.js

```diff
@@ -0,0 +1,6 @@
+export { default as AttributeSetsTransform } from "./attribute-sets-transform.js";
+export { default as CSSOpenTypeTransform } from "./css-font-open-type-transform.js";
+export { default as CSSSetsFormatter } from "./css-sets-formatter.js";
+export { default as DataJsonFormatter } from "./data-json-formatter.js";
+export { default as JsonSetsFormatter } from "./json-sets-formatter.js";
```

Comment:
Do we need both `JsonSetsFormatter` and `DataJsonFormatter`? It looked like we removed `JsonSetsFormatter` from `style-dictionary.config.js`. Was it replaced in favor of `DataJsonFormatter` in the config, or are we missing it in the config?

Reply:
Yes great point, we aren't using it.